praktikumsplaner-frontend/frontend/src/components/TheSnackbar.vue

```diff
@@ -28,7 +36,8 @@ const defaultTimeout = 5000;
 const show = ref(false);
 const timeout = ref(defaultTimeout);
 const message = ref("");
-const color = ref("info");
+const backgroundColor = ref("info");
+const btnTextColor = ref("#DA8CFF");
```

Comment:
pls use the vuetify theme settings to setup colors

Reply:
was this comment already addressed it does not look like it was, or is my understanding of themes and their usage wrong